src/app/state/account/saga.ts

```diff
@@ -17,6 +17,17 @@ import { selectAccountAddress } from './selectors'
 export function* fetchAccount(action: PayloadAction<string>) {
   const address = action.payload
 
+  if (!isValidAddress(address)) {
+    yield* put(
+      actions.accountError({
+        code: WalletErrors.InvalidAddress,
+        message: 'Invalid address.',
+      }),
+    )
+    yield* put(actions.setLoading(false))
```

Comment:
loading was not initiated at this point so we don't need to dispatch this action

Reply:
That's what I thought, too, but in fact the stats _is_ initialized with loading set to true, see here: https://github.com/oasisprotocol/oasis-wallet-web/blob/94ab4f055cbd43a57f241ccc9b7ca82d69a8188c/src/app/state/account/index.ts#L17

So if loading fails, we do need to set it to false somehow. (I have tested it, and the loading dialog stays without this line.)

Now it's good question why is it initialized like that. I don't know, maybe it's to avoid some flickering when initially loading the code? It has been changed from false to true by @Esya here: https://github.com/oasisprotocol/oasis-wallet-web/commit/dc226b682889bdf9798378cb9c038ede573b6c78#diff-dfe080912ab14112544da12f1bc8e10274d9392417a4edc981d0cc504cb7c98e

Anyway I didn't want to mess with the state, so I just cleared it here.

Reply:
At this point, `store.getState().account.loading` is still true, because it has been initialized that way.
The loading dialog is invisible because it's normally rendered by the account page, which is now hidden, but still, the internal state of the app is erroneously marked as loading. So I think it makes sense to set loading to false.